test/unit_tests/braket/aws/test_aws_session.py

```diff
@@ -849,7 +849,7 @@ def test_get_default_jobs_role(aws_session, job_role_arn, job_role_name):
                 ],
                 "IsTruncated": False,
             },
-            {"Marker": "resp-marker"},
```

Comment:
nit: Could you please update the list_roles response in the test data to accurately reflect the path prefix ?